components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/Context.java

```diff
@@ -97,6 +86,21 @@ public Response proceed() throws Exception {
   }
 
   //
+  // Attributes
+  //
+
+  /**
+   * Custom attributes to configure backing and logging.
+   */
+  public static class Attributes
```

Comment:
This change appears unrelated to the PR title "request response builders" - any particular reason to make this public? Also, since `AttributesMap`'s default constructor already creates a new hash map, why not use that?
```
public static class Attributes
    extends AttributesMap
{
   // custom attributes go here?
}
```